go/vt/vttablet/tabletserver/tabletenv/config.go

```diff
@@ -148,6 +148,7 @@ func registerTabletEnvFlags(fs *pflag.FlagSet) {
 	fs.BoolVar(&currentConfig.AnnotateQueries, "queryserver-config-annotate-queries", defaultConfig.AnnotateQueries, "prefix queries to MySQL backend with comment indicating vtgate principal (user) and target tablet type")
 	fs.BoolVar(&currentConfig.WatchReplication, "watch_replication_stream", false, "When enabled, vttablet will stream the MySQL replication stream from the local server, and use it to update schema when it sees a DDL.")
 	fs.BoolVar(&currentConfig.TrackSchemaVersions, "track_schema_versions", false, "When enabled, vttablet will store versions of schemas at each position that a DDL is applied and allow retrieval of the schema corresponding to a position")
+	fs.Int64Var(&currentConfig.SchemaVersionMaxAgeSeconds, "schema_version_max_age_seconds", 0, "max age of schema version records to keep in memory when schema tracking is enabled")
```

Comment:
We should use dashes rather than underscores for new flags as we're in the process of moving all flags to dashes.

Does this apply to the query serving schema tracker or only to the vreplication historian? I thought that it was only the latter but the help output makes it seem like the former.

Reply:
> Does this apply to the query serving schema tracker or only to the vreplication historian? I thought that it was only the latter but the help output makes it seem like the former.

Definitely the vreplication historian. I get the language confused for the different "schema tracking" that is done by both mechanisms. I'll update the help description.